src/bun.js/bindings/napi.cpp

```diff
@@ -341,6 +342,7 @@ class NAPIFunction : public JSC::JSFunction {
         NAPICallFrame frame(JSC::ArgList(args), function->m_dataPtr);
 
         auto scope = DECLARE_THROW_SCOPE(vm);
+        v8::HandleScope handleScope(v8::Isolate::fromGlobalObject(static_cast<Zig::GlobalObject*>(globalObject)));
```

Comment:
I think this can technically be a node:vm global object but I don't think we handle this correctly elsewhere either
```suggestion
        v8::HandleScope handleScope(v8::Isolate::fromGlobalObject(static_cast<Zig::GlobalObject*>()));
```